### PR TITLE
feat(Gate-Pool): Deploy all helm charts with umbrella structure

### PR DESCRIPTION
**Pull request by fabiodmota**
Change Chart Structure for umbrella helm charts
issue https://github.com/eclipse-tractusx/bpdm/issues/139

### CODE REVIEW

File: charts/bpdm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    repository: catenax-ng/tx-bpdm/gate
+    pullPolicy: Always
+    tag: ""
+  # ... more configuration for bpdm-gate if need...
+  applicationConfig:
+    bpdm:
+      pool:
+        base-url: ""
+
+bpdm-pool:
```
Comment (nicoprow) [inline charts/bpdm/values.yaml:15]: I know due to limitations of helm currently child of child dependencies are ignored. But I feel for being more clear here we should still disable postgres and opensearch dependencies here by setting enabled: false

File: charts/bpdm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+bpdm-gate:
```
Comment (nicoprow) [inline charts/bpdm/values.yaml:1]: I know due to limitations of helm currently child of child dependencies are ignored. But I feel for being more clear here we should still disable postgres dependency here by setting enabled: false

File: charts/bpdm/values.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  enabled: true
+  replicaCount: 1
+  image:
+    registry: ghcr.io
+    repository: catenax-ng/tx-bpdm/pool
+    pullPolicy: Always
+    tag: ""
+  opensearch:
+    masterService: ""
+    fullnameOverride: "bpdm-pool-opensearch"
```
Comment (nicoprow) [inline charts/bpdm/values.yaml:25]: Is opensearch naming needed here? I feel since the dependency is not created should be not needed, right?

**Review by nicoprow — CHANGES_REQUESTED** [review]
Some minor clarifications needed

File: charts/bpdm/values.yaml (hunk 2 above)
Comment (fabiodmota) [reply charts/bpdm/values.yaml:1]: done

File: charts/bpdm/values.yaml (hunk 3 above)
Comment (fabiodmota) [reply charts/bpdm/values.yaml:28]: for now as it stands we have a need to have it like this , we will take a look further on this with the help of dev sec team

**Review by nicoprow — APPROVED** [review]

**Pull request merged**